File: com/pdfjet/Alignment.java
/*
 * Alignment.java
 *
 * Copyright (c) 2026 PDFjet Software
 * Licensed under the MIT License. See LICENSE file in the project root.
 */
package com.pdfjet;

/**
 * Used to specify the horizontal and vertical alignment, for example of the
 * text in a Cell, TextBox, TextBlock, Paragraph or TextColumn.
 */
public enum Alignment {
    /** Aligns to the left. */
    LEFT,
    /** Aligns to the right. */
    RIGHT,
    /** Centers horizontally or vertically. */
    CENTER,
    /** Justifies the text. */
    JUSTIFY,
    /** Aligns to the top. */
    TOP,
    /** Aligns to the bottom. */
    BOTTOM;
}

File: net/pdfjet/Alignment.cs
/*
 * Alignment.cs
 *
 * Copyright (c) 2026 PDFjet Software
 * Licensed under the MIT License. See LICENSE file in the project root.
 */

namespace PDFjet.NET {
    /// <summary>
    /// Used to specify the horizontal and vertical alignment, for example of the
    /// text in a Cell, TextBox, TextBlock, Paragraph or TextColumn.
    /// </summary>
    public enum Alignment {
        /// <summary>Aligns to the left.</summary>
        LEFT,
        /// <summary>Aligns to the right.</summary>
        RIGHT,
        /// <summary>Centers horizontally or vertically.</summary>
        CENTER,
        /// <summary>Justifies the text.</summary>
        JUSTIFY,
        /// <summary>Aligns to the top.</summary>
        TOP,
        /// <summary>Aligns to the bottom.</summary>
        BOTTOM
    }
}
